modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -4340,6 +4370,20 @@ private static File _testNotExists(File dir, String fileName) {
 		return file;
 	}
 
+	private static void _testPropertyKeyExists(Path path, String key)
+		throws Exception {
+
+		try (InputStream inputStream = Files.newInputStream(path)) {
```

Comment:
We already have a `FileTestUtil.readProperties` method which reads from the classpath, why not adding an overload that receives a `Path`?